admin_manual/installation/installation_wizard.rst

```diff
@@ -170,8 +178,13 @@ and files:
 * The :file:`apps/` directory should be owned by ``[HTTP user]:[HTTP group]``
 * The :file:`config/` directory should be owned by ``[HTTP user]:[HTTP group]``
 * The :file:`themes/` directory should be owned by ``[HTTP user]:[HTTP group]``
+* The :file:`assets/` directory should be owned by ``[HTTP user]:[HTTP group]``
 * The :file:`data/` directory should be owned by ``[HTTP user]:[HTTP group]``
 * The :file:`[ocpath]/.htaccess` file should be owned by ``root:[HTTP group]``
 * The :file:`data/.htaccess` file should be owned by ``root:[HTTP group]``
 * Both :file:`.htaccess` files are read-write file owner, read-only group and 
   world
+
+If you want to update your installation via the GUI or the occ-command,
+check :ref:`setting_strong_permissions` in the Update section.
```

Comment:
There isn't an Update section, so you should remove 'in the Update section'. The ref automatically prints the section title, 'Setting Strong Permissions'.